src/plugins/intel_gna/src/optimizer/gna_pass_manager.cpp

```diff
@@ -1290,7 +1290,9 @@ void InsertConcatAligningFilterPass::run() {
             // correcting offset by copy layer insertion. This can be improved by collapsing copy and affine or diagonal
             // later-on if next concat inputs requires align filter - then current input also requires either copy or
             // align filter
-            if (ALIGN64(offset) != offset || (ALIGN64(outputSize) != outputSize && useAlignFilterIf(input_idx + 1))) {
+            if (ALIGN(offset, Limitations::get_instance()->get_memory_alignment()) != offset ||
```

Comment:
I would hide all usages of ALIGN in methods of Limitations.
It would decouple a bit this part of code.

Reply:
Once the above usage was replaced with the is_aligned function, there are now only two occurrences of ALIGN(..., Limitations::get_instance()->get_memory_alignment()) remaining. I would like to leave as it is.